start-shop

```diff
@@ -12,13 +12,76 @@ load_dramatically ()
     sleep 2
 }
 
-while getopts ":b" opt; do
+load_even_more_dramatically ()
```

Comment:
Would be cool if you could write this more elegantly and iterate through an array of tuples. Perhaps not the most straightforward thing to attempt in bash though. First thought that comes to mind is that you could use a plain test file as input, and specify the loading as such:
```
1 .
1 .
1 .
0.25 .
0.25 .
4 *i
```
Then we could start building a library of loading animations (!) and write a pretty simple bit of bash to consume those scripts and print out the characters.

Reply:
ah thats legit. Ill see if i can figure it out